userguide/content/en/docs/Quickstart/_index.md

```diff
@@ -0,0 +1,89 @@
+
+---
+title: "Quickstart"
```

Comment:
I think as it is, this is slightly confusing - we have "Quickstart" and "Getting started" as sort of peer pages and it's not clear which one you should go to first (I also have a personal dislike of the term Quickstart but I can live with it. :) ). We also should make it clear that you can use the Docker image as a real environment for working locally on your own sites, not just as a "kick the tyres with the example site" tool for a tutorial.

How about we move this under /getting_started/ and rename it "Quickstart with Docker" (so it'll show up as a sub-page in the nav)

I would also in the main Getting started add a new section under Prerequisites and installation (right at the top so users who don't want to have to install anything can see it) - something like

### Use our Docker image

We provide a [Docker](linky) image that you can use to run and test your Docsy site locally, without having to install all Docsy's dependencies.

You can see how to get started with this approach by following our Quickstart tutorial (linky). If you don't want to use Docker, follow the instructions below to install Hugo and PostCSS.

Reply:
I like your suggestions. I just reviewed the [docker-compose PR](https://github.com/google/docsy-example/pull/93) and suggested a couple of changes, as soon as we get that merged I will update this PR.

Reply:
Would be great to get this in now that the Docker update is merged! *subtle hint face*

Reply:
Currently working on it :)